### DISCUSSION TITLE
cn/2022/01/content-based-image-retrieval/

### DISCUSSION THREAD

**Discussion by giscus** [leovan/leovan.me#37] [Comments]
# cn/2022/01/content-based-image-retrieval/

本文主要参考自 SIFT meets CNN: A decade survey of instance retrieval 1 和 Deep Learning for Instance Retrieval: A Survey 2。 基于内容的图像检索（Content-based image retrieval，CBIR），属于图

https://leovan.me/cn/2022/01/content-based-image-retrieval/

**Comment by duduOliver** [comment]
您好！请问文中 ***基于内容的图像检索里程碑节点图*** 是什么工具做的？

**Reply by leovan** [reply]
是 macOS 下的一款软件 omnigraffle

**Reply by duduOliver** [reply]
好的，非常感谢！可惜我用Win 😂

**Reply by leovan** [reply]
windows 下用 visio 画应该 OK 的，因为 macOS 下没有 visio 我才用 omnigraffle 的😂

**Reply by duduOliver** [reply]
好的！Win上面Visio，亿图也能做，只是我感觉 omnigraffle 更符合我的审美，也可能只是我之前没怎么接触过 omnigraffle，所以有个新鲜感😂
